sdk/daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/interpretation/Error.scala

```diff
@@ -148,6 +148,14 @@ object Error {
 
     sealed abstract class Error extends Serializable with Product
 
+    // TODO: https://github.com/DACH-NY/canton-network-utilities/issues/2916: introduce structured CCTP error reporting
+    //  sealed case class CCTP(error: CCTP.Error) extends Error
+    //  object CCTP {
+    //    sealed abstract class Error extends Serializable with Product
+    //    final case class InvalidKeyError(msg: String) extends Error
+    //    final case class SignatureError(msg: String) extends Error
+    //  }
```

Comment:
For the moment, we simply return false rather than reporting a more structured error. This is done to avoid to reduce the changes blast radius.